CDS/scripts/js_utils/youtube/youtube-upload-lib.js

```diff
@@ -98,19 +99,27 @@ function addPosterImageIfExists(connection, videoId, youtubeClient, account) {
         }
 
         dataStore.get(connection, 'meta', 'poster_image').then(posterImage => {
-            downloadPosterImage(posterImage.value, `/tmp/${videoId}.jpg`).then(filename => {
-                const payload = {
-                    videoId: videoId,
-                    onBehalfOfContentOwner: account,
-                    media: {
-                        body: fs.createReadStream(filename)
-                    }
-                };
-
-                youtubeClient.thumbnails.set(payload, (err, result) => err ? reject(err) : resolve(result));
-            }).catch(err => {
-                reject(err);
-            });
+            downloadPosterImage(posterImage.value, `${POSTER_IMAGE_DOWNLOAD_DIR}/${videoId}.jpg`)
```

Comment:
does `createReadStream` care about file extension? Presumably it reads it as bytes? I might just omit it entirely...